pkg/key/cluster.go

```diff
@@ -0,0 +1,25 @@
+package key
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
+)
+
+const (
+	ingressControllerConfigMapName = "ingress-controller-values"
+	nginxIngressControllerAppName  = "nginx-ingress-controller-app"
+)
+
+func ClusterConfigMapName(customResource v1alpha1.App) string {
+	// A separate config map is used for Nginx Ingress Controller.
+	if AppName(customResource) == nginxIngressControllerAppName {
+		return ingressControllerConfigMapName
+	}
```

Comment:
Nginx IC is an exception. I think its better to have it here than in app-admission-controller.